server/webapp/WEB-INF/rails/config/routes.rb

```diff
@@ -256,8 +256,6 @@
 
       get 'dashboard', controller: :dashboard, action: :dashboard, as: :show_dashboard
 
-      match 'version', controller: :version, action: :show, as: :version, via: %w(get head)
```

Comment:
I don't think the `head` was intentional. I suppose this support was removed, which is OK.

Reply:
s/intentional/unintentional/

Reply:
intentional : we don't need the meta information in this case in the http head